kuma/users/templates/users/email/welcome/html.ltxt

```diff
@@ -56,6 +56,7 @@
             </li>
         </ul>
         <p>
+          {# Note: If you change this string also change ``WELCOME_EMAIL_STRINGS`` in kuma/users/tasks.py #}
```

Comment:
No, I think `{% trans %}` is correct here and should work. I removed it from the plain text since `{% trans %}` collapses whitespace which isn't what we wanted in a plaintext email -- we want the whitespace.